scripts/apps/search/directives/SearchParameters.js

```diff
@@ -28,7 +28,12 @@ export function SearchParameters($location, asset, tags, metadata, meta, common,
                 scope.query = params.q;
                 scope.flags = false;
                 scope.common = common;
-                scope.meta = meta = {};
+                scope.meta = meta;
+                for (var member in meta) {
```

Comment:
I might be wrong though, either way, this is very weird. In case my suggestion above doesn't work, it might be more obvious to do:
```js
Object.keys(meta).forEach(key => delete meta[key]);
```